html-aam/index.html

```diff
@@ -580,7 +580,10 @@ <h4 id="el-area-no-href">`area` <span class="el-context">(no `href` attribute)</
             </tr>
             <tr>
               <th>Comments</th>
-              <td></td>
+              <td>
+                User agents MAY still expose an `a` element lacking a `href` attribute with a `link` role in the event an author specifies interactive behavior for the element. E.g., by using an
+                `onclick` attribute.
```

Comment:
Would it help to distinguish this as an "event handler attribute" (to distinguish from other content attributes) and cover other types of intended functionality?

Reply:
yes. that's a better idea than this one example.